src/aspire/source/xform.py

```diff
@@ -283,11 +349,14 @@ def _apply_xform(xform, im, indices, adjoint=False):
     A simple global function (i.e. not a method) that is capable of being cached by joblib.Memory object's `cache`
     method.
     """
+    # Extract the short xfrom name for the long string
+    # xform_name = str(xform).split()[0].split('.')[-1]
+    xform_name = xform.__class__.__name__
     if not adjoint:
-        logger.info('  Applying ' + str(xform))
+        logger.info('  Applying ' + xform_name)
```

Comment:
Shouldn't this stay as `str(xform)` since we now have #232 that should take care of this?

Reply:
Yes.  `str(xform)` was used in the new commit 3c064d0.